doc/topics/jinja/index.rst

```diff
@@ -1700,6 +1700,23 @@ Will insert the following message in the minion logs:
 
 .. jinja_ref:: custom-execution-modules
 
+Python methods
+====================
+
+A power feature of jinja that is only hinted at in the official jinja
```

Comment:
This I have to disagree with you. I think powerful sounds better, and if you don't like the style the whole word could just be dropped. I think "A feature" sounds better than "A power feature".